sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/TelemetryPartA.cs

```diff
@@ -18,6 +18,7 @@ namespace Azure.Monitor.OpenTelemetry.Exporter
     /// </summary>
     internal class TelemetryPartA
     {
+        private const string DateTimeFormat = "yyyy-MM-dd HH:mm:ss.ffffZ";
```

Comment:
This doesn't seem to follow https://github.com/Azure/azure-sdk-for-net/blob/6a5198bc6d982a9f6b29f4a26d443dcb68f174ef/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Generated/Models/TelemetryItem.cs#L41.

Consider `yyyy-MM-ddTHH:mm:ss.fffffffZ`.

Reply:
Fixed in commit 2c2adf41ae